pkg/storage/stores/indexshipper/shipper.go

```diff
@@ -96,6 +99,33 @@ func (cfg *Config) Validate() error {
 	return storage.ValidateSharedStoreKeyPrefix(cfg.SharedStoreKeyPrefix)
 }
 
+func (cfg *Config) GetUniqueUploaderName() (string, error) {
```

Comment:
nit: Can we small example how does the file name would look like as a comment description for this function?

Reply:
Filenames are handled by the underlying index builders i.e `boltdb-shipper` and `tsdb`. I will add some comments at appropriate places.